public/app/features/plugins/admin/components/PluginDetailsPage.tsx

```diff
@@ -111,6 +111,7 @@ export const getStyles = (theme: GrafanaTheme2) => {
     tabContent: css`
       overflow: auto;
```

Comment:
Is this `overflow:auto` needed? We have a custom scrollbar for the page-inner which should be all that's required for scrolling the page. 🤔

Reply:
@leventebalogh added it in his previous commit. I am not sure what is the use case for it. If we can remove it would be the best. @leventebalogh do you remember why ?

Reply:
Phew, good question. I added this comment above it `// Needed due to block formatting context`, but unfortunately I don't remember what was the actual reason. Probably it's not needed anymore?